### DISCUSSION TITLE
Can I disable the blinking cursor in pdf-view mode?

### DISCUSSION THREAD

**Discussion by brabalan** [meow-edit/meow#558] [Q&A] [answered]
I use pdf-tools to view pdfs, and with meow I see a thin line blinking all around the pdf page, which I assume is the cursor. Is there a way to disable it?

In doom-emacs, they also seem to have to do it:
https://github.com/doomemacs/doomemacs/blob/master/modules/tools/pdf/config.el#L63

```emacs-lisp
(setq-hook! 'pdf-view-mode-hook evil-normal-state-cursor (list nil))
```

**Comment by DogLooksGood** [comment]
```
(make-variable-buffer-local 'meow-cursor-type-normal)
(add-hook 'pdf-view-mode-hook (lambda () (setq-local meow-cursor-type-normal nil)))
```

**Reply by brabalan** [reply, marked as answer]
Thank you for the suggestion. I could not make this work (even replacing `normal` with `motion`), so what I ended up doing is disabling cursor blinking (`(blink-cursor-mode 0)`), which is what was actually bothering me.
